files/zh-cn/web/api/audiolistener/forwardy/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "AudioListener: forwardY 属性"
+slug: Web/API/AudioListener/forwardY
+---
+
+{{ APIRef("Web Audio API") }}
+
+`forwardY` 只读属性属于 {{ domxref("AudioListener") }} 接口，它是一个 {{domxref("AudioParam")}}，表示定义监听者指向的前进方向的方向向量的 y 值。
+
+> **注意：** 当与 {{domxref("PannerNode")}} 一起使用时，如果 {{domxref("PannerNode.panningModel", "panningModel")}} 设置为 equalpower，则参数为 _a-rate_；否则为 _k-rate_。
+
+## 值
+
+一个 {{domxref("AudioParam")}}。其默认值为 0，范围可在正负无穷之间。
+
+## 示例
+
+查看 [BaseAudioContext.createPanner()](/zh-CN/docs/Web/API/BaseAudioContext/createPanner#examples) 以获取示例代码。
```

Comment:
Please update the hash, as we could point to the correct section when we translated `BaseAudioContext.createPanner()`.

```suggestion
查看 [BaseAudioContext.createPanner()](/zh-CN/docs/Web/API/BaseAudioContext/createPanner#示例) 以获取示例代码。
```